saleor/order/models.py

```diff
@@ -292,9 +298,14 @@ class OrderedItem(models.Model, ItemLine):
     unit_price_gross = models.DecimalField(
         pgettext_lazy('OrderedItem field', 'unit price (gross)'),
         max_digits=12, decimal_places=4)
+    order = models.ForeignKey(Order, related_name='items')
```

Comment:
What is this? An item belongs to exactly one group that belongs to exactly one order.